pimd/pim_instance.h

```diff
@@ -194,6 +198,8 @@ struct pim_instance {
 	int64_t nexthop_lookups;
 	int64_t nexthop_lookups_avoided;
 	int64_t last_route_change_time;
+
+	uint64_t gm_rx_drop_sys;
```

Comment:
This information is not added in the show CLI, is this missed or intentional and will be taken care in separate PR.

Reply:
It's intentional, I haven't decided yet where to show it — because the socket was per interface before, it was possible to count it on the interface, but now it is per VRF and we don't know which interface it belongs on. And there's no VRF show command yet for MLD.